.github/workflows/test.yml

```diff
@@ -20,6 +20,14 @@ jobs:
       with:
         path: src/github.com/containerd/nerdctl
         fetch-depth: 25
```

Comment:
Can we just increase fetch-depth here?

Reply:
You could pick any arbitrary number, it'll always eventually be wrong.

I personally use a different approach for my own workflows, but it's more complicated than what you have (which is broken) or what I'm proposing (which will solve the PR case, but wouldn't technically solve a Push event with a similarly long set of commits.

Reply:
> You could pick any arbitrary number, it'll always eventually be wrong.

100 should be enough for everyone.

If you really need to fetch all commits you can just remove `fetch-depth` line, but I don't see necessity of doing that.